Explorer/Assets/Scripts/Utility/ParcelMathHelper.cs

```diff
@@ -89,6 +89,10 @@ public static ParcelCorners CalculateCorners(Vector2Int parcelPosition)
         public static Vector2Int FloorToParcel(Vector3 position) =>
             new (Mathf.FloorToInt(position.x / PARCEL_SIZE), Mathf.FloorToInt(position.z / PARCEL_SIZE));
 
+        public static Vector2Int ToParcel(this Vector3 position) =>
```

Comment:
Can you use the method FloorToParcel instead of adding the extension? So we don't have the duplicated logic

Or, on the other hand, maybe we can make all of the above usages use the extension?